packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -87,6 +93,27 @@ const HeaderTools = ({
                 document.querySelector('html').classList.toggle('pf-v5-theme-dark')} />
             </ToolbarItem>
           )}
+          {hasRTLSwitcher && (
+            <ToolbarItem>
+              <ToggleGroup aria-label="Example direction toggle">
+                <ToggleGroupItem
+                  icon={isRtl ? <AlignRightIcon /> : <AlignLeftIcon />}
+                  aria-label="Toggle direction"
+                  buttonId="ws-rtl-toggle"
+                  isSelected={false}
+                  onChange={() => setIsRtl(isRtl => {
+                    const examples = document.querySelectorAll('.ws-example');
+                    examples.forEach(ex => ex.classList.toggle('pf-v5-m-dir-rtl'));
+                    return !isRtl;
+                  })}
+                />
+              </ToggleGroup>
+              <Switch id="ws-rtl-switch" label="Example direction" defaultChecked={false} onChange={() =>{
+                const examples = document.querySelectorAll('.ws-example');
+                examples.forEach(ex => ex.classList.toggle('pf-v5-m-dir-rtl'));
```

Comment:
Not a big deal, but I don't think there's value in switching the example header (`.ws-example-header`). I _think_ we could target `.ws-preview` (or `.ws-example .ws-preview` if we need to scope it?)? If not, we could add the `.pf-v5-m-dir-ltr` class to the example header, which will keep it from flipping.